lib/companies/jobs.ex

```diff
@@ -18,7 +18,7 @@ defmodule Companies.Jobs do
 
   """
   def all do
-    Repo.all(Job)
+    Job |> Ecto.Query.where(is_deleted: false) |> Repo.all()
```

Comment:
Could you multiple these pipelines (ln 21, 39) to match the rest of the codebase?